### PR TITLE
Istio - add service-level metrics to integration dashboard

### PR DESCRIPTION
**Pull request by johnbryan**
Istio exposes service-level metrics (https://istio.io/latest/docs/reference/config/metrics/) as well as envoy proxy metrics. But we only displayed the envoy metrics. This adds the service-level metrics.

### CODE REVIEW

File: dashboards/istio/istio-envoy-prometheus.png
Comment (yqlu) [inline dashboards/istio/istio-envoy-prometheus.png:1]: Let's update the screenshot to reflect the text widget color changes you made.

File: integrations/istio/prometheus_metadata.yaml [hunk 1]
```diff
@@ -8,6 +8,22 @@ platforms:
...
+        kind: CUMULATIVE
+        value_type: DISTRIBUTION
+      - name: prometheus.googleapis.com/istio_request_bytes/histogram
+        prometheus_name: istio_request_bytes
+        kind: CUMULATIVE
+        value_type: DISTRIBUTION
+      - name: prometheus.googleapis.com/istio_response_bytes/histogram
+        prometheus_name: istio_response_bytes
+        kind: CUMULATIVE
+        value_type: DISTRIBUTION
```
Comment (yqlu) [inline integrations/istio/prometheus_metadata.yaml:26]: Just double-checking if intentional... the dashboard doesn't showcase `istio_request_bytes` and `istio_response_bytes`

File: dashboards/istio/istio-envoy-prometheus.png
Comment (johnbryan) [reply dashboards/istio/istio-envoy-prometheus.png:1]: updated

File: integrations/istio/prometheus_metadata.yaml (hunk 1 above)
Comment (johnbryan) [reply integrations/istio/prometheus_metadata.yaml:26]: Yes, not featured in the dashboard, but these are collected by the integration.

**Comment by johnbryan** [comment]
Added units on the new charts, which were missing.

**Review by yqlu — APPROVED** [review]

**Pull request merged**